src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
If the ws connection disconnects/reconnects, or the client has a transfer that is past expiry, they can query the ledger to get the transfer state.